test/controllers/GuidanceControllerSpec.scala

```diff
@@ -27,15 +28,16 @@ import play.api.test.Helpers._
 import uk.gov.hmrc.play.bootstrap.tools.Stubs.stubMessagesControllerComponents
 import uk.gov.hmrc.http.SessionKeys
 import forms.SubmittedAnswerFormProvider
-import models.{PageEvaluationContext, PageContext}
-import models.ocelot.{Process, ProcessJson, Phrase,KeyedStanza, Page => OcelotPage}
-import models.ocelot.stanzas.{Question => OcelotQuestion,_}
-import models.ui.{CurrencyInput => UiCurrencyInput,_}
+import models.{PageContext, PageEvaluationContext}
+import models.ocelot.{KeyedStanza, Phrase, Process, ProcessJson, Page => OcelotPage}
```

Comment:
Rearranging the order to alphabetic order; that's the kind of OCD update I do!

Reply:
Thats the sort of OCD update Intellij does :)